src/js/structs/ServicesList.js

```diff
@@ -24,7 +22,7 @@ module.exports = class ServicesList extends List {
 
     if (filters) {
       if (filters.ids) {
-        services = _.filter(services, function (service) {
+        services = services.filter(function (service) {
           return this.ids.indexOf(service.id) !== -1;
         }, {ids: filters.ids});
```

Comment:
yeah i think we can. i'm not too sure why it was implemented like that in the beginning